api-2.0/src/test/java/org/openmrs/module/attachments/AttachmentsServiceTest.java

```diff
@@ -50,8 +55,8 @@ public void tearDown() throws IOException {
 		testHelper.tearDown();
 	}
 	
-	protected static List<Attachment> asSortedAttachments(List<Obs> obsList) {
```

Comment:
@mks-d  i removed the static qualifier on  `ComplexDataHelper complexDataHelper;` however i had to remove the static qualifier in asSortedAttachments method because i could not make a static reference to a non-static field `complexDataHelper` .
Unfortunately the tests in this class fail on running them with all the changes with the `javaNullPointerException` ,i looked into the stack trace and it points back to the constructor 	on this line in the Attachment constructor 	`setBytesMimeType(complexDataHelper.getContentType(obs.getComplexData()));` ,

Reply:
@mks-d please have a look at this ...Thanks